src/wdio/services/TerraCommands/chai-methods.js

```diff
@@ -20,6 +33,24 @@ function accessible() {
     `expected no accessibility violations but got:\n\t${errors[0]}`,
     'expected accessibility errors but received none',
   );
+
+  // The block of code below is used to display a warning when a test
```

Comment:
We are waiting on logging this in a different PR right? if so, want to remove.

Reply:
Logged https://github.com/cerner/terra-toolkit/issues/370 and removed the commented code.